maintainer/conda/environment.yml

```diff
@@ -29,7 +29,11 @@ dependencies:
   - tidynamics>=1.0.0
   - tqdm>=4.43.0
   - sphinxcontrib-bibtex
+  - mdaencore
+  - waterdynamics
   - pip:
+      - mdahole2
+      - pathsimanalysis
```

Comment:
Thanks for adding all these, looks as if they should have been there all along.

Quick question: where do we use conda/environment.yml ?

Reply:
This is only used for RTD docs building.